rholang/src/test/scala/coop/rchain/rholang/interpreter/matcher/StreamTSpec.scala

```diff
@@ -155,44 +152,78 @@ class StreamTLawsSpec
 
   implicit def eqT: Eq[Throwable] = Eq.allEqual
 
-  checkAll(
-    "StreamT.MonadLaws",
-    MonadTests[StreamTEffect].monad[Int, Int, String]
-  )
-  checkAll(
-    "StreamT.AlternativeLaws",
-    AlternativeTests[StreamTEffect].alternative[Int, Int, String]
-  )
-  checkAll(
-    "StreamT.MonadTransLaws",
-    MonadTransTests[StreamT].monadTrans[Effect, Int, String]
-  )
-  checkAll(
-    "StreamT.MonadLayerControlLaws",
-    MonadLayerControlTests[StreamTEffect, Effect, Stream].monadLayerControl[Int, String]
-  )
-  checkAll(
-    "StreamT.MonadErrorLaws",
-    MonadErrorTests[StreamTEffect, String].monadError[Int, Int, String]
-  )
-  checkAll(
-    "StreamT.MonadErrorUnitLaws",
-    MonadErrorTests[StreamTEffect, Unit].monadError[Int, Int, String]
-  )
-
-  checkAll("StreamT.SyncLaws", SyncTests[StreamTEffect].sync[Int, Int, String])
-
-  checkAll(
-    "StreamT.SyncLaws", {
-      val fromEffect =
-        λ[Effect ~> StreamTEffect[?]](
-          e => StreamT.liftF(e)
-        )
-      BracketTests[StreamTEffect[?], Throwable].bracketTrans[Effect, Int, Int](
-        fromEffect
+  test("StreamT.MonadLaws") {
+    for {
+      (testName, prop) <- MonadTests[StreamTEffect].monad[Int, Int, String].props
+      _                = info(testName)
```

Comment:
Is this needed? What does it do?

Reply:
Also in case we can't get `CatsSuite` back, please remove the redundancy. The tests should look similar to the ones we had in terms of succinctness.

Reply:
> Is this needed? What does it do?

If you are talking about `info`, it is just for more info when problems occur.

Reply:
So what happened is that in this PR: https://github.com/typelevel/cats/pull/2970 `CatsSuite` was removed - " it was never really intended for external usage". I think that we should just replace it with hand-rolled equivalent. Apart from pulling all the cats implicits (which we should do selectively anyway) it was adding the:
```scala
trait CatsSuite
    extends AnyFunSuiteLike
    with Matchers
    with ScalaCheckDrivenPropertyChecks
    with Discipline
    with TestSettings
//...
    with StrictCatsEquality {

  implicit override val generatorDrivenConfig: PropertyCheckConfiguration =
    checkConfiguration

  // disable Eq syntax (by making `catsSyntaxEq` not implicit), since it collides
  // with scalactic's equality
  override def catsSyntaxEq[A: Eq](a: A): EqOps[A] = new EqOps[A](a)
// ...
}
```

I think it is easily replaceable